src/NuGet.Services.AzureSearch/Db2AzureSearch/Db2AzureSearchCommand.cs

```diff
@@ -100,7 +105,11 @@ private async Task ExecuteAsync(CancellationToken token)
                 // Push all package package data to the Azure Search indexes and write the version list blobs.
                 var allOwners = new ConcurrentBag<IdAndValue<IReadOnlyList<string>>>();
                 var allDownloads = new ConcurrentBag<DownloadRecord>();
-                await PushAllPackageRegistrationsAsync(cancelledCts, produceWorkCts, allOwners, allDownloads);
+
+                var storageResult = await _auxiliaryFileClient.LoadExcludedPackagesAsync(etag: null);
+                var excludedPackages = storageResult.Data ?? new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Can `LoadExcludedPackagesAsync` really return null data?

Reply:
not really. habituated with null checks 😆